docs/contributing/testing-principles.md

```
@@ -192,3 +239,20 @@ test('fetches from a disposable server', async () => {
 	expect(await response.text()).toBe('ok')
 })
 ```
+
+## Nightly cleanup
+
+A scheduled Cursor automation (Keep Tests Tight) re-reads this page and removes
+tests that do not meet the bar. Prompt for that automation:
+
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)